docs/src/openapi-v1.yaml

```diff
@@ -242,13 +242,25 @@ paths:
         required: false
         schema:
           type: boolean
+          default: false
       - name: extrinsicDocs
         in: query
         description: When set to `true`, every extrinsic will have an extra `docs`
           property with a string of the extrinsics documentation.
         required: false
         schema:
           type: boolean
+          default: false
+      - name: noAuthor
```

Comment:
I don't really feel super strongly about this, but it seems a little like a double-negative to call this `noAuthor` and then set the default to `false`. What if we called the param `authorId` and said the default value was `true`? I don't know...like I said, just a thought.

Reply:
Great point, will change 👍